src/js/actions/ActionsCreators.js

```diff
@@ -280,8 +279,11 @@ export function setPayout(payload) {
     if (typeof payout === 'undefined') {
       payout = getState().getIn(['storage', 'payout']);
     }
+    else if (isNaN(payout*1) || !Number.isInteger(payout*1) || parseInt(payout) < 0 || parseInt(payout) > 100) {
```

Comment:
@qingweibinary but what if it's not undefined? if it's `'10.1'` it's not undefined so it doesn't become `+` and is allowed